management-service/src/main/java/pl/pwr/zpi/cluster/dto/ClusterConfigurationDTO.java

```diff
@@ -45,6 +47,22 @@ public static ClusterConfigurationDTO ofCluster(ClusterConfiguration clusterConf
                                 .map(NodeConfigurationDTO::ofNodeConfiguration)
                                 .toList()
                 )
+                .updatedAtMillis(clusterConfiguration.getUpdatedAtMillis())
+                .build();
+    }
+
+    public static ClusterConfigurationDTO defaultConfiguration() {
+        return ClusterConfigurationDTO.builder()
+                .accuracy(Accuracy.HIGH)
+                .isEnabled(true)
+                .running(true)
+                .generatedEveryMillis(0L)
+                .slackReceivers(Collections.emptyList())
+                .discordReceivers(Collections.emptyList())
+                .emailReceivers(Collections.emptyList())
+                .applicationConfigurations(Collections.emptyList())
+                .nodeConfigurations(Collections.emptyList())
+                .updatedAtMillis(0L)
```

Comment:
souldn't be now() here?

Reply:
No, if the default is used it means that cluster configuration wasn't created and also wasn't updated, so the frontend should display "Never"